.github/workflows/translate.yml

```diff
@@ -60,7 +60,8 @@ jobs:
 
       - name: update-repo-before-commit
         run: |
-          git submodule sync
+          rm -rf ios-sdk
+          git submodule deinit ios-sdk
```

Comment:
hmm can't we just skip the submodule stuff?

Reply:
it was already removed before your comment